app/MMB/src/components/header/Header.js

```diff
@@ -21,6 +21,7 @@ export default class Header extends React.Component<Props> {
     return (
       <View style={styleSheet.wrapper}>
         <StatusBar data={booking} />
+        <View style={styleSheet.separator} />
```

Comment:
We have quite a few of these now I think. Does it make sense to make a component, or are they all different somehow?

Reply:
Will do. We have basically 2 and this one for iOS is actually similar to the separator for Android.

Reply:
Done... 🙂